andela/arakari-rc
# #139284023 CRUD operations SPM

#### What does this PR do?
 - Fixes getting started video bug
 - Adds implementation of (not-so) static pages. Create pages in backend, have them appear in front

 #### Description of Task to be completed?
Create and edit static pages, such as Contact, About, or Support.
Implement static pages for a given shop.
Enable admins to customize the look on the static page content

 #### How should this be manually tested?
- Log in as admin
- Add page
- Check navigation bar for the route to the page

 #### Any background context you want to provide?

 #### What are the relevant pivotal tracker stories?
#139284023 

 #### Screenshots (if appropriate)
![image](https://cloud.githubusercontent.com/assets/24469235/24539917/7369254c-15f9-11e7-8785-5018ac703d86.png)
![image](https://cloud.githubusercontent.com/assets/24469235/24586246/a82b571c-17a4-11e7-8e47-26f8bd4d1e4e.png)
![image](https://cloud.githubusercontent.com/assets/24469235/24586258/d6da716a-17a4-11e7-8476-1d450e47b131.png)

 #### Questions:
